addons/mail/static/src/components/attachment_list/attachment_list.xml

```diff
@@ -2,16 +2,16 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.AttachmentList" owl="1">
-        <div class="o_AttachmentList" t-attf-class="{{ className }}" t-ref="root">
+        <div class="o_AttachmentList d-flex flex-column justify-content-start" t-attf-class="{{ className }}" t-ref="root">
             <t t-if="attachmentList">
-                <div class="o_AttachmentList_partialList o_AttachmentList_partialListImages">
+                <div class="o_AttachmentList_partialList o_AttachmentList_partialListImages d-flex flex-grow-1 flex-wrap">
                     <t t-foreach="attachmentList.attachmentImages" t-as="attachmentImage" t-key="attachmentImage.localId">
-                        <AttachmentImage className="'o_AttachmentList_attachment'" localId="attachmentImage.localId" onAttachmentRemoved="props.onAttachmentRemoved"/>
+                        <AttachmentImage className="'o_AttachmentList_attachment my-1 me-1 mw-100'" localId="attachmentImage.localId" onAttachmentRemoved="props.onAttachmentRemoved"/>
```

Comment:
The spacing is not the same vertically and horizontally, it looks a bit strange?
![image](https://user-images.githubusercontent.com/38213504/151147930-15b11184-6542-4772-9db6-ce75fe5705d8.png)

Same issue whether it is image or non image.

Reply:
Yep, I didn't change it, It's the same on master.

Vertical margins are both `.25rem`, while horizontal ones are `0` for start and `.25rem` for end.

As a result the grid gap is `.5rem` vertically (`.25rem` + `.25rem`) and `.25rem` horizontally (`.25rem` +`0`).
I thought to fix this by setting `margin-inline-end` to `.5rem`, but it will likely move to the next line an attachment when it's not necessary.

Since this grid must be refactored anyway, I decided to mirror master's value for now.
I'll [REF] this grid when will be the time to tickle the design.

Reply:
Ok fair enough to improve it separately, thank you for the check.